api/cpp/include/slint.h

```diff
@@ -607,6 +607,37 @@ class Window
         inner.dispatch_pointer_event(event);
     }
 
+#ifdef SLINT_FEATURE_EXPERIMENTAL
```

Comment:
Why is it guarded with the experimental #ifdef when all three "functions" are stable API in Rust?

Reply:
The implementation in C++ is in api/cpp/platform.rs that's only compiled in when feature is on. also the cbindgen is ran over this file only when the feature is on.

Reply:
Ah, so the plan is to remove the #ifdef when the implementation is moved out of experimental?